src/server.c

```diff
@@ -6429,6 +6429,12 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                             "# Cluster\r\n"
                             "cluster_enabled:%d\r\n",
                             server.cluster_enabled);
+    }
+
+    /* Cluster Status */
+    if (all_sections || (dictFind(section_dict, "cluster_status") != NULL)) {
+        if (sections++) info = sdscat(info, "\r\n");
+        info = sdscatprintf(info, "# Cluster Status\r\n");
```

Comment:
I would prefer `cluster_info` as that's the original command, easier to the remember the subsection.

```suggestion
    /* Cluster Info */
    if (all_sections || (dictFind(section_dict, "cluster_info") != NULL)) {
        if (sections++) info = sdscat(info, "\r\n");
        info = sdscatprintf(info, "# Cluster Info\r\n");
```

Reply:
Yes. I thought of that. It just seemed weird that we have a section called "cluster_info" in the "INFO" command :)

Reply:
Will leave it to you to decide. Personally, I would prefer the symmetry.

Reply:
I will change it